packages/lit-element/src/lit-element.ts

```diff
@@ -128,6 +128,7 @@ export class LitElement extends ReactiveElement {
    * @category lifecycle
    */
   connectedCallback() {
+    this.renderOptions.isConnected = true;
```

Comment:
A user could disconnect the element in their `connectedCallback` before calling `super.connectedCallback` and the state would then be wrong. This is a pretty corner case so not sure it matters, but alternatively, we could check `isConnected` directly at render time.

Reply:
You're suggesting to hit the DOM binding for every element's first render?

Reply:
At least in native, adding this to update doesn't seem to show up, so I guess we'll go with that...?
```ts
    if (!this.hasUpdated) {
      this.renderOptions.isConnected = this.isConnected;
    }
 ```
![image](https://user-images.githubusercontent.com/1354932/130393681-148a53f4-5a6b-41ed-b7f3-69d2ff12b307.png)